packages/vue/package.json

```diff
@@ -6,8 +6,7 @@
   "module": "dist/index.js",
   "exports": {
     ".": "./dist/index.js",
-    "./styles.css": "./dist/style.css",
-    "./dist/style.css": "./dist/style.css"
```

Comment:
@wlee221 Not sure why was re-exported – the paths are the same, but we also don't want `/dist` to become common.